src/Tickets/Blocks/Ticket/app/editor/dashboard/container.js

```diff
@@ -16,12 +17,74 @@ import Template from './template';
 import { actions, selectors } from '@moderntribe/tickets/data/blocks/ticket';
 import { withStore } from '@moderntribe/common/hoc';
 
-const getIsConfirmDisabled = ( state, ownProps ) => (
-	! selectors.isTicketValid( state, ownProps ) ||
-		! selectors.getTicketHasChanges( state, ownProps ) ||
-		selectors.isTicketDisabled( state, ownProps ) ||
-		selectors.getTicketHasDurationError( state, ownProps )
-);
+/**
+ * Filters whether the confirm button should be disabled.
+ *
+ * @since TBD
+ *
+ * @param {Object} state    The state of the store.
+ * @param {Object} ownProps The own props of the component.
+ *
+ * @return {boolean} Whether the confirm button should be disabled.
+ */
+const shouldConfirmBeDisabled = (state, ownProps) => {
+	/**
+	 * Filters whether the ticket is valid.
+	 *
+	 * @since TBD
+	 *
+	 * @param {boolean} isValid  Whether the ticket is valid.
+	 * @param {Object}  state    The state of the store.
+	 * @param {Object}  ownProps The own props of the component.
+	 */
+	const isTicketValid = applyFilters(
```

Comment:
This filter will apply before the default logic, which works for the SLR case where the determination is based on Layout  and Seat Type only, but might not work for other filtering functions.

I would apply the default logic first, then apply the filter on the result of the default logic:

```js
const getIsConfirmDisabled = (state) => {
	let isConfirmDisabled = (
		!selectors.getRSVPTempTitle(state) ||
		!selectors.getRSVPHasChanges(state) ||
		selectors.getRSVPIsLoading(state) ||
		selectors.getRSVPHasDurationError(state)
	);
	
	isConfirmDisabled = applyFilters( ... );
	
	return isConfirmDisabled;
};
```

Reply:
I have added a filter for that `tec.tickets.blocks.confirmButton.isDisabled` but i think for this specific case the one being utilized is better.

The reason is that it's filtering whether or not the Ticket on its current state is valid or not, rather than if the confirm button should be disabled or not which can't be decided only by checking the ticket's validity.

Now, the `getTicketHasChanges` method in combination with the filtered `isTicketValid` are going to produce the deisred results :D

Reply:
Have ignored that in your example you are using RSVP methods instead of tickets. I have assumed that its simply a typo.